include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -626,6 +638,24 @@ namespace Portable
     unsigned int
     get_mg_level() const;
 
+    /**
+     * Return the cell iterator given the index within the color.
+     *
+     * @param index The index within the color
+     * @param color The color index
+     * @param dof_handler_index Index of the DoFHandler (default 0)
+     */
+    typename DoFHandler<dim>::cell_iterator
+    get_cell_iterator(const unsigned int index,
+                      const unsigned int color,
```

Comment:
I would switch the order of the parameter. First the color and then the index within the color